specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-04-01-preview/machineLearningServices.json

```diff
@@ -1228,6 +1228,284 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/Microsoft.MachineLearningServices/workspaces/{workspaceName}/linkedWorkspaces": {
+      "get": {
+        "tags": [
+          "LinkedWorkspaces"
+        ],
+        "x-ms-examples": {
+          "ListLinkedWorkspaces": {
+            "$ref": "./examples/listLinkedWorkspaces.json"
+          }
+        },
+        "operationId": "LinkedWorkspaces_List",
+        "description": "List linked workspaces under a workspace per some query criteria.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "name": "link-subscriptionId",
+            "in": "query",
+            "type": "string",
+            "description": "The subscriptionid of the linked workspace."
+          },
+          {
+            "name": "link-resource-group",
+            "in": "query",
+            "type": "string",
+            "description": "The resource group name of the linked workspace."
+          },
+          {
+            "name": "link-resource-name",
+            "in": "query",
+            "type": "string",
+            "description": "The resource name of the linked workspace."
+          },
+          {
+            "name": "link-resourceId",
+            "in": "query",
+            "type": "string",
+            "description": "The resourceId of the linked workspace."
+          },
+          {
+            "name": "link-type",
+            "in": "query",
+            "type": "string",
+            "description": "The type of the linked workspace.",
+            "enum": [
+              "Synapse"
+            ],
+            "x-ms-enum": {
+              "name": "LinkType",
+              "modelAsString": true
+            }
+          },
+          {
+            "name": "uai-resourceId",
+            "in": "query",
+            "type": "string",
+            "description": "The resourceId of the user assigned identity for the linked workspace."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the linked workspaces.",
+            "schema": {
+              "type": "array",
+              "description": "array of linked workspaces.",
+              "items": {
+                "$ref": "#/definitions/LinkedWorkspace"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      },
+      "put": {
```

Comment:
This is an illegal operation per ARM, as it is a PUT against a collection. The PUT should be on the path below:
`/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/Microsoft.MachineLearningServices/workspaces/{workspaceName}/linkedWorkspaces/{linkName}`

Reply:
will update